spirv_glsl.cpp

```diff
@@ -804,6 +804,9 @@ bool CompilerGLSL::ssbo_is_std430_packing(const SPIRType &type)
 
 string CompilerGLSL::layout_for_variable(const SPIRVariable &var)
 {
+	if (is_legacy_es() || (!options.es && options.version < 330))
```

Comment:
It's taking some time getting my head around the implications of (optionally) disabling the 420 pack, so I've modified the condition to handle the obvious case for now. 

In that case invalid GLSL was emitted, so this is an improvement even if it doesn't tackle the problem of >= 130 and no 420 pack available (e.g. OS X)